README.md

```diff
@@ -243,4 +242,41 @@ On MacOS or Windows systems, we recommend either using a VM or the provided `.de
 - `build`: builds and tags `envbuilder:latest` for your current architecture.
 - `develop`: runs `envbuilder:latest` against a sample Git repository.
 - `test`: run tests.
-- `test-registry`: stands up a local registry for caching images used in tests.
\ No newline at end of file
+- `test-registry`: stands up a local registry for caching images used in tests.
+
+<!--- Code generated by docsgen. DO NOT EDIT. --->
+<!--- START docsgen --->
+## Environment Variables
+
+| Environment variable | Default | Description |
+| - | - | - |
+| SETUP_SCRIPT |  | SetupScript is the script to run before the init script. It runs as the root user regardless of the user specified in the devcontainer.json file. SetupScript is ran as the root user prior to the init script. It is used to configure envbuilder dynamically during the runtime. e.g. specifying whether to start systemd or tiny init for PID 1. |
+| INIT_SCRIPT | sleep infinity | InitScript is the script to run to initialize the workspace. |
```

Comment:
I think it would look nice if these were inline code blocks, like this:
```suggestion
| `INIT_SCRIPT` | `sleep infinity` | InitScript is the script to run to initialize the workspace. |
```
(For "Default", we'd only insert `` if it's a non-empty value.)

Reply:
Done!